### DISCUSSION TITLE
FOSSDEM 2024

### DISCUSSION THREAD

**Discussion by fmartingr** [go-shiori/shiori#831] [Announcements]
Hey folks, a quick one! 

This weekend is [FOSDEM](https://fosdem.org/2024/) in Belgium, Brussels, and I will be attending with some friends. I'm not organizing anything nor giving any talks, **but** I will be carrying _stickers_ in the remote possibility that any Shiori user is around and want to chat.

You will spot me easily since I'm going to be the only one with the Shiori logo in his backpack. Probably!

![IMG_8204](https://github.com/go-shiori/shiori/assets/812088/4ff32a4a-84c4-4d8d-9b52-1554365f7119)

See you there!

**Comment by Monirzadeh** [comment]
I wish I could be there. Maybe someday :)
Have fun.
